RefApp/ViewControllers/LoginViewController.swift

```diff
@@ -0,0 +1,173 @@
+//
+//  LoginViewController.swift
+//  RefApp
+//
+//  Created by Udaya Sri Senarathne on 2018-11-19.
+//  Copyright © 2018 emp. All rights reserved.
+//
+
+import UIKit
+import Exposure
+
+class LoginViewController: UIViewController {
+    
+    var environment: Environment!
+    
+    let topHolderView: UIView = {
+        let view = UIView()
+        return view
+    }()
+    
+    let bottomHolderView: UIView = {
+        let view = UIView()
+        return view
+    }()
+    
+    let logoImageView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.image = UIImage(named: "redbee")
+        imageView.contentMode = .scaleAspectFill
+        return imageView
+    }()
+    
+    let usernameTextField: RBMTextField = {
+        let textfield = RBMTextField(placeHolderText: NSLocalizedString("Username", comment: ""))
+        return textfield
+    }()
+    
+    let passwordTextField: RBMTextField = {
+        let textfield = RBMTextField(placeHolderText: NSLocalizedString("Password", comment: ""))
+        textfield.isSecureTextEntry = true
```

Comment:
and

textfield.textContentType = UITextContentType.password

Reply:
👍